.travis.yml

```diff
@@ -3,12 +3,30 @@ go:
   - 1.7
   - tip
 
+install:
+  - wget https://github.com/google/protobuf/releases/download/v3.1.0/protoc-3.1.0-linux-x86_64.zip
+  - unzip protoc-3.1.0-linux-x86_64.zip -d /tmp
+  - sudo cp /tmp/bin/protoc /usr/bin/protoc
+  - sudo chmod 777 /usr/bin/protoc
+  - rm protoc-3.1.0-linux-x86_64.zip
```

Comment:
Yeah, I think it is. The `wget` is literally what the Travis CI docs list in their example for protobuf.